pkg/patch/cmd.go

```diff
@@ -21,15 +21,18 @@ import (
 )
 
 type patchArgs struct {
-	appImage      string
-	reportFile    string
-	patchedTag    string
-	workingFolder string
-	buildkitAddr  string
-	timeout       time.Duration
-	ignoreError   bool
-	format        string
-	output        string
+	appImage               string
+	reportFile             string
+	patchedTag             string
+	workingFolder          string
+	buildkitAddr           string
```

Comment:
Instead of explicitly defining these addr/cert args here, can use `buildkit.Opts` and set the flags to use `&bkopts.<Field>`

Reply:
done :D